trade-finance-manager-ui/templates/utilisation-reports/check-keying-data.njk

```diff
@@ -0,0 +1,59 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% import "./_macros/selected-reported-fees-details-table.njk" as selectedReportedFeesDetailsTable %}
+{% import "./_macros/check-keying-data-table-row.njk" as checkKeyingDataTableRow %}
+
+{% extends "index.njk" %}
+
+{% set backLink = {"href": "/utilisation-reports/" + reportId} %}
+
+{% block pageTitle %}Check keying data{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row govuk-!-padding-top-7">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l">{{ bank.name }}, {{ formattedReportPeriod }}</span>
+      Check before generating keying sheet data
+    </h1>
+
+    {% if numberOfMatchingFacilities === 1 %}
+      <h1 class="govuk-heading-m">{{ numberOfMatchingFacilities }} matched facility</h1>
+    {% else %}
+      <h1 class="govuk-heading-m">{{ numberOfMatchingFacilities }} matched facilities</h1>
+    {% endif %}
+
+    <table class="govuk-table premium-payments-table" data-module="moj-sortable-table" data-cy="check-keying-data-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Facility ID</th>
+          <th scope="col" class="govuk-table__header">Exporter</th>
+          <th scope="col" class="govuk-table__header govuk-table__header--numeric">Reported fees</th>
+          <th scope="col" class="govuk-table__header govuk-table__header--numeric">Reported payments</th>
+          <th scope="col" class="govuk-table__header govuk-table__header--numeric">Payments received</th>
+          <th scope="col" class="govuk-table__header">Status</th>
```

Comment:
The design shows these as sortable, possibly need to raise on the ticket what 'standard' sorting means payments received column which can have multiple values? (see my other comments re grouping which obviously impacts the ability to sort other columns)

Reply:
I've made every column sortable aside from payments received (due to them being listed in specific cases)